src/iotjs.c

```diff
@@ -26,6 +26,7 @@
 #endif
 #include "jerryscript-port.h"
 #include "jerryscript.h"
+#include "jerryscript-ext/handle-scope.h"
```

Comment:
Also, we are not starting using HandleScope yet, so adding this by MACRO would be reasonable :)

Reply:
Once NAPI module are introduced to ShadowNode, handle-scope are necessary rather than configurable.